src/framework/express.ts

```diff
@@ -20,7 +20,7 @@ export const createExpressApp = (): express.Express => {
   });
 
   app.use((req: Request, res: Response, next) => {
-    const labels = { path: req.path, status: res.statusCode, method: req.method };
+    const labels = { path: req.route.path, status: res.statusCode, method: req.method };
```

Comment:
sure this is always available? what happens when no route has been matched (404s etc)